modules/nw-ovn-k-baseline-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: Concept
+[id="BaselineAdminNetworkPolicy"_{context}"]
+= BaselineAdminNetworkPolicy
+
+`BaselineAdminNetworkPolicy` (BANP) is a cluster-scoped custom resource definitions (CRDs). As a {product-title} administrator, you can use BANP to setup and enforce optional baseline network policies that are overridable by users using `NetworkPolicies` if need be. Rule actions for BANP are `allow` or `deny`.
```

Comment:
Same changes as requested for ANP: https://github.com/openshift/openshift-docs/pull/70800#discussion_r1482995038
```suggestion
`BaselineAdminNetworkPolicy` (BANP) is a cluster-scoped custom resource definition (CRD). As a {product-title} administrator, you can use BANP to setup and enforce optional baseline network policie rules that are overridable by users using `NetworkPolicies` if need be. Rule actions for BANP are `allow` or `deny`.
```
Add https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2091-admin-network-policy#story-5-cluster-wide-default-guardrails to this aspect so that users know why BANP is present in addition to ANP.
there are two technical aspects here:

1. it can act as the catch-all for passed traffic
2. it can act as the default cluster security guardrail

Also worth mentioning at a given time there can only be 1 BANP in the cluster. It is a singleton object.

Reply:
This information is below. I'll move it up.